src/main/resources/templates/tracing/contactList.html

```diff
@@ -9,31 +9,86 @@
 <div layout:fragment="content">
 
     <div class="border border-secondary p-2 mb-3">
-        <h5 class="mb-4">Kontaktverfolgung</h5>
+        <h4 class="mb-4">Kontaktverfolgung</h4>
 
-        <div th:if="${#lists.isEmpty(tableValues)}">
+        <div th:if="${numberOfContacts}  == 0">
             <p>Für die angegebene Mail-Adresse konnten keine Kontakte gefunden werden.</p>
         </div>
 
-        <div th:unless="${#lists.isEmpty(tableValues)}">
+        <div th:unless="${numberOfContacts} == 0">
 
             <p>Hier sind die Kontakte f&uuml;r <span th:text="${target}"></span>.
-                Zur Weitergabe an das Gesundheitsamt können die Daten <a
-                        th:href="@{download(email=${target})}">hier
-                    als Excel-Datei</a>
-                heruntergeladen werden.</p>
-
-            <table class="table table-hover mt-3">
-                <thead class="table-info">
-                <tr>
-                    <th th:each="header: ${tableHeaders}" th:text="${header}"/>
-                </tr>
-                </thead>
-
-                <tr th:each="row: ${tableValues}">
-                    <td th:each="cell: ${row}" th:text="${cell}"/>
-                </tr>
-            </table>
+                Zur Weitergabe an das Gesundheitsamt können die Daten in Form von Excel Dateien heruntergeladen werden.
+                Dabei kann jeweils eine seperate Excel-Datei für Studenten, Lehrende/Hochschulangestellte und Gäste
+                heruntergeladen werden oder alle Kontaktpersonen gesammelt in einer Exceldatei.
```

Comment:
I can't see a link for all contacts in one file? Remove this part of the sentence?

Reply:
Yes, removed it